src/picai_prep/dcm2mha.py

```diff
@@ -36,6 +37,8 @@ def __init__(
         input_path: PathLike,
         output_path: PathLike,
         settings_path: PathLike,
+        verify_dicom_filenames: bool = True,
+        scan_postprocess_func: Optional[Callable[[sitk.Image], sitk.Image]] = None,
```

Comment:
Request to minimize the number of complicated non-obvious parameters of the constructor, and instead plop these in a **kwargs. so `kwargs.get('verify_dicom_filenames', True)` instead

Reply:
Furthermore, if put in a `self.kwargs` for example, these can be consumed at an arbitrary point during processing

Reply:
Let's tackle this in the Item refactoring?